program-test/tests/warp.rs

```diff
@@ -289,6 +289,7 @@ async fn stake_rewards_bench_550_k() {
     stake_rewards_filter_bench_core(550_000, false).await;
 }
 
+// jwash: why filter here?
```

Comment:
I refactor this from an existing test for filtering stake by 1 sol. For the benchmark test, we want all the accounts not be filtered.